src/dodal/devices/apple2_undulator.py

```diff
@@ -138,22 +173,70 @@ async def set(self, value: T) -> None:
         await self.set_move.set(value=1, timeout=timeout)
         await wait_for_value(self.gate, UndulatorGateStatus.CLOSE, timeout=timeout)
 
-    @abc.abstractmethod
-    async def _set_demand_positions(self, value: T) -> None:
-        """Set the demand positions on the device without actually hitting move."""
 
-    @abc.abstractmethod
-    async def get_timeout(self) -> float:
-        """Get the timeout for the move based on an estimate of how long it will take."""
+class MotorWithoutStop(Motor):
+    """A motor that does not support stop."""
 
-    async def raise_if_cannot_move(self) -> None:
-        if await self.fault.get_value() != 0:
-            raise RuntimeError(f"{self.name} is in fault state")
-        if await self.gate.get_value() == UndulatorGateStatus.OPEN:
-            raise RuntimeError(f"{self.name} is already in motion.")
+    def __init__(self, prefix: str, name: str = ""):
+        super().__init__(prefix=prefix, name=name)
+        del self.motor_stop  # Remove motor_stop from the public interface
+
+    async def stop(self, success=False):
+        LOGGER.info(f"Stopping {self.name} is not supported.")
```

Comment:
```suggestion
    async def stop(self, success: bool = False):
        LOGGER.info(f"Stopping {self.name} is not supported.")
```

nit: Type hints

From discussion IRL, we think that this doesn't need to raise an exception although JW would prefer if it did. The plan should be allowed to continue running when the device has been asked to stop, because stopping the InsertionDevice before the move is finished causes the device to lose home and need to recalibrate the whole beamline.